aries-site/src/pages/templates/navigation.mdx

```diff
@@ -1,32 +1,35 @@
-export const page = getPageDetails(title);
-export const relatedContent = getRelatedContent(title);
-
 export const navPatterns = [
-  'Hub and Spoke Navigation',
-  'Persistent Navigation',
+  'Drill Down Navigation',
+  'Side to Side Navigation',
+  'Reverse Navigation',
+  'Matrix Navigation',
 ];
 export const navCards = navPatterns.map(pattern => getPageDetails(pattern));
-export const globalSidebar = getPageDetails('Global Sidebar');
-export const routerExample = getPageDetails('React Router');
-
-View navigation patterns and templates for HPE applications, learn how to incorporate Global Sidebar into applications living in the HPE as-a-service ecosystem, and see how to implement leading React routing packages with navigation templates.
 
-## Patterns
+Navigation is the overall structure that provides users screen direction to complete a task logically and easily through a website or an application. 
 
-Recommended patterns, templates, and examples for use within the context of a single application, website, or subsection within an application. Plus, demonstration for how local navigation patterns, such as [Hub & Spoke](/templates/hub-and-spoke-navigation) and [Persistent Navigation](/templates/persistent-navigation), may be combined with the [Global Sidebar](/extend/global-sidebar) for easy maneuverability across applications within the HPE ecosystem.
+Part of creating a seamless appplication or interface is creating an obvious flow from one view/interaction to the next. Think of Navigation as a story map where you help guide your user through each process. 
 
-### Navigation Within an Application
+Consider the following flow below where a user is navigated from the HPE.com home page to a Sign Up page to create an account to the view of their dashboard. 
 
-Flexible in nature, the following patterns can be applied to present contextual, supplemental, and hierachical needs within an application or subsection of a larger application.
+## Types of  Navigation
 
-- [Hub & Spoke patterns](/templates/hub-and-spoke-navigation) lend themselves well to applications primarily concerned with task execution.
-- Reach for [Persistent Navigation](/templates/persistent-navigation) when a user needs orientation or easy access to application sections.
+There are 4 different types of navigational directions a user can maneuver in:
 
 <CardGrid cards={navCards} />
+
+## What makes a successful Navigation
+
+Users should always be able to tell where they are on a page, application or product and should be predictable as to how they can get to the next step or destination. A task has a start and an end point so it is important to keep the functionality or task(s) in mind when creating products with common user experiences and expectations.
+
+**Follow the points below for a successful Navigation:**
+- Identify the problem
+- Understand what the user’s role is and common processes followed
+- Who is this being intended for and determine the most common tasks
```

Comment:
Since the other bullet points start with verbs, we should phrase this one to start with a verb as well. It seems to be the same as the second bullet point (unless I'm misunderstanding), so maybe we can remove the third bullet point all together?

Reply:
@vavalos5

Reply:
Agreed @taysea